### PR TITLE
Homography fixes

### PR DESCRIPTION
**Pull request by dcelik**
This PR fixes the cloud side issues with uploadHomography:
 * uploadHomography no longer takes or saves camera images
 * uploadHomography has become configHomography route
 * Status enums have been changed to reflect this
 * misc cleanups in uploadHomography

### CODE REVIEW

File: app/handlers/configHomography.py [hunk 1]
```diff
@@ -10,17 +9,15 @@
...
-class UploadHomographyHandler(BaseHandler):
+class ConfigHomographyHandler(BaseHandler):
     """
-    @api {post} /uploadHomography/ Upload Homography
-    @apiName UploadHomography
-    @apiVersion 0.1.0
+    @api {post} /configHomography/ Config Homography
+    @apiName ConfigHomography
+    @apiVersion 0.2.0
     @apiGroup Upload
```
Comment (runnersaw) [inline app/handlers/configHomography.py]: Should this still be Upload? Should upload and config be merged? Should we make it clearer that this needs to be done.

File: app/handlers/configHomography.py [hunk 2]
```diff
@@ -10,17 +9,15 @@
...
-    @apiDescription Use this route to upload homography files for a project.
+    @apiDescription Use this route to upload homography data for a project.
 
-    @apiParam {String} identifier The identifier of the project to upload files to.
-    @apiParam {File} aerial An aerial photo of the intersection.
-    @apiParam {File} camera A screenshot of the intersection from the video.
+    @apiParam {String} identifier The identifier of the project for which to configure the homography.
     @apiParam {Integer} unit_pixel_ratio The unit_pixel_ratio of the images (ie. 0.05 meters per pixel).
     @apiParam {JSON} aerial_pts A JSON array containing the coordinates of point clicks on the aerial image as arrays in the form [x_coord, y_coord]
```
Comment (runnersaw) [inline app/handlers/configHomography.py]: I would say the type is Array, not JSON (JSON just stands for JavaScript Object Notation). Maybe JSON Array if you'd rather be more specific.

File: app/handlers/configHomography.py [hunk 3]
```diff
@@ -31,43 +28,25 @@ class UploadHomographyHandler(BaseHandler):
...
+        StatusHelper.set_status(self.identifier, Status.Type.CONFIG_HOMOGRAPHY, Status.Flag.COMPLETE)
         self.finish("Upload Homography")
 
     def write_homography_files(self):
         project_dir = get_project_path(self.identifier)
         aerial_pts = self.get_body_argument('aerial_pts')
         camera_pts = self.get_body_argument('camera_pts')
+
         homography, mask = cv2.findHomography(\
                             np.array(literal_eval(camera_pts)),\
```
Comment (runnersaw) [inline app/handlers/configHomography.py]: Glad that we're actually using the safe version and not `eval` here :)

Comment (runnersaw) [reply app/handlers/configHomography.py]: Sidenote: https://www.reddit.com/r/Python/comments/hftnp/ask_rpython_recovering_cleared_globals/ is fascinating. Especially the part where the guy figures out how to recover cleared `__builtins__`

**Review by runnersaw — APPROVED** [review]
Awesome! 🚢 it

File: app/handlers/configHomography.py (hunk 1 above)
Comment (dcelik) [reply app/handlers/configHomography.py]: good catch! i forgot to change that but maybe that is something we would want to change later

File: app/handlers/configHomography.py (hunk 3 above)
Comment (dcelik) [reply app/handlers/configHomography.py]: Yup want to make sure we don't have any issues here :) Looking at the documentation, this actually might need to be try/catched to make sure we grab any malformed data exceptions.

File: app/handlers/configHomography.py (hunk 2 above)
Comment (dcelik) [reply app/handlers/configHomography.py]: Technically its an Array of tuples but converting it to JSON is how its understood. I will change this to JSON Array to be specific.

**Comment by dcelik** [comment]
Updated according to Sawyer's comments.

**Pull request merged**